extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/SmallRyeOpenApiProcessor.java

```diff
@@ -159,6 +160,22 @@ OpenApiFilteredIndexViewBuildItem smallryeOpenApiIndex(CombinedIndexBuildItem co
                         new OpenApiConfigImpl(ConfigProvider.getConfig())));
     }
 
+    @BuildStep
+    public List<FriendlyJaxRsAnnotationPrefixBuildItem> registerJaxRsSupportedAnnotation() {
+        List<FriendlyJaxRsAnnotationPrefixBuildItem> prefixes = new ArrayList<>();
+        prefixes.add(new FriendlyJaxRsAnnotationPrefixBuildItem(
+                DotName.createSimple("org.eclipse.microprofile.openapi.annotations.tags")));
+        prefixes.add(new FriendlyJaxRsAnnotationPrefixBuildItem(
+                DotName.createSimple("org.eclipse.microprofile.openapi.annotations.servers")));
+        prefixes.add(new FriendlyJaxRsAnnotationPrefixBuildItem(
+                DotName.createSimple("org.eclipse.microprofile.openapi.annotations.security")));
+        prefixes.add(new FriendlyJaxRsAnnotationPrefixBuildItem(
+                DotName.createSimple("org.eclipse.microprofile.openapi.annotations.media")));
+        prefixes.add(new FriendlyJaxRsAnnotationPrefixBuildItem(
+                DotName.createSimple("org.eclipse.microprofile.openapi.annotations.extensions")));
```

Comment:
Why not adding the whole `org.eclipse.microprofile.openapi.annotations` package?

Reply:
The implementation of the detection of "allowed" annotation was not able to detect the root package of an annotation, only the package of the annotation itself, that's why I included all package of every OpenApi annotations.
I improved the detection to declare the root package of a set of annotations like OpenAPI